bootstrap/lib/server.js

```diff
@@ -24,7 +26,10 @@ const initHandler = (execute, checkHealth = notImplementedHealthCheck) => () =>
         .status(HTTP_ERROR_UNSUPPORTED_MEDIA_TYPE)
         .send(HTTP_ERROR_UNSUPPORTED_MEDIA_TYPE_MESSAGE)
     }
-
+    req.body.data = Object.assign(
+      req.body.data === undefined ? {} : req.body.data,
+      parseQueryString(req.query),
+    )
```

Comment:
`Object.assign()` modifies an object in place. So we should either do:

```javascript
req.body.data = req.body.data || {}
Object.assign(req.body.data, parseQueryString(req.query))
```

or just use the spread operator:

```javascript
req.body.data = { ...req.body.data, parseQueryString(req.query) }
```

Because of [optional spreading](https://stackoverflow.com/questions/46957194/javascript-es6-spread-operator-on-undefined) feature, I suggest going with option #2.